ui/tempo-plugin/index.ts

```diff
@@ -0,0 +1,10 @@
+import { TempoTraceQuery } from '././src/plugins/TempoTraceQuery';
```

Comment:
this file is not under the folder `src`, I guess we can remove it ?

Reply:
You're right, this should not have been in the push. Thanks for catching it, the file was removed.